### PR TITLE
lowercasing function scoped variables

### PR DESCRIPTION
**Pull request by dltacube**
- [x] lowercasing variables local to functions

https://github.com/mautrix/imessage/pull/189#discussion_r1494294407

### CODE REVIEW

File: imessage/bluebubbles/api.go [hunk 1]
```diff
@@ -1397,13 +1397,13 @@ func (bb *blueBubbles) convertBBContactToiMessageContact(bbContact *Contact) (*i
...
 	case string:
-		// ID is already a string, use it as is
-		convertedID = ID
+		// id is already a string, use it as is
+		convertedID = id
 	case int:
-		// ID is an integer, convert it to a string
-		convertedID = strconv.Itoa(ID)
+		// id is an integer, convert it to a string
+		convertedID = strconv.Itoa(id)
```
Comment (tulir) [inline imessage/bluebubbles/api.go:1406]: Note: parsing JSON into `any` will turn numbers into `float64`, not `int`

Comment (trek-boldly-go) [reply imessage/bluebubbles/api.go:1406]: Thanks for catching this tulir. @dltacube can you drop an error logger in the default case so we see it fail, then add another case for the float type?

Comment (dltacube) [reply imessage/bluebubbles/api.go:1406]: Yea, and wow, good catch.

Comment (dltacube) [reply imessage/bluebubbles/api.go:1406]: I left the structs alone and kept the `any` fields intact and did what you suggested: added a `float64` conversion along with logging an error message in the default case.

Comment (trek-boldly-go) [reply imessage/bluebubbles/api.go:1406]: @dltacube were you able to test and see this properly convert?

Comment (dltacube) [reply imessage/bluebubbles/api.go:1406]: @trek-boldly-go i built it and it's running on my machine. I'll mark the PR as ready once I confirm that it's working correctly.

Comment (dltacube) [reply imessage/bluebubbles/api.go:1406]: For me it's coming back as a string already. Do you know off the top of your head if this is coming from the matrix API or the bb API? I feel like contacts could potentially come from both but I need to dig a bit.

Also looking at this note here and trying to think of ways to replicate this. Also if that's true then the conversion to int wouldn't have worked either, so either this user hasn't seen another number or it's still occasionally crashing for them. https://github.com/mautrix/imessage/blob/590dcb04864248a2fab79d0285f8fb50dfe78b16/imessage/bluebubbles/interface.go#L118

Is this user https://github.com/rollingonchrome by any chance?

**Review by trek-boldly-go — APPROVED** [review]

**Pull request merged**